quinn-proto/src/connection/mod.rs

```diff
@@ -3165,9 +3165,9 @@ impl RttEstimator {
         }
     }
 
-    /// Get current smoothed RTT estimate
-    fn get(&self) -> Duration {
-        self.smoothed.unwrap_or(self.latest)
+    pub fn get(&self) -> Duration {
+        self.smoothed
+            .map_or(self.latest, |x| cmp::max(x, self.latest))
```

Comment:
Ah, I was confused about where that `get` was coming from in the other issue. I think the version introduced in #822 is more correct, however. The only time I see this `max` in the current draft is in [computing the time threshold for loss detection](https://quicwg.org/base-drafts/draft-ietf-quic-recovery.html#name-time-threshold), which we already apply separately.

Reply:
Ah, that was an unintentional conflict resolution error. Fixed now.